src/components/NcEmojiPicker/NcEmojiPicker.vue

```diff
@@ -391,47 +395,55 @@ export default {
 		},
 
 		afterShow() {
-			// add focus trap in modal
-			const picker = this.$refs.picker
-			picker.$el.addEventListener('keydown', this.checkKeyEvent)
-
-			// set focus on input search field
-			const input = this.$refs.search?.$refs.inputField?.$refs.input
-			if (input) {
-				input.focus()
-			}
+			this.$refs.search.focus()
 		},
 
 		afterHide() {
```

Comment:
Can we focus whatever in trigger slot on close, to imitate focus-trap?

Reply:
But it's not working in current state, so should be fine to follow-up separately